src/engine/ruleset/integrations/windows/rules/defense-evasion-disabling-windows-logs.yml

```diff
@@ -0,0 +1,60 @@
+name: rule/defense-evasion-disabling-windows-logs/0
+
+metadata:
+  module: Windows
+  title: Disable Windows Event and Security Logs Using Built-in Tools
+  description: Identifies attempts to disable EventLog via the logman Windows utility, PowerShell, or auditpol. This is often done by
+    attackers in an attempt to evade detection on a system.
+    this method to execute malicious scripts and avoiding writing it to disk.
+  versions: [Vista, "7", "8", "10", "11", Server 2012, Server 2016, Server 2019, Server 2022]
+  compatibility: This rule has been tested on Wazuh version 4.5.
+  author:
+    name: Wazuh, Inc.
+    date: 2023/09/15
+  references:
+    - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/logman
+    - https://medium.com/palantir/tampering-with-windows-event-tracing-background-offense-and-defense-4be7ac62ac63
+
+check: >-
+    array_contains($event.type, start) AND array_contains($event.category, process)
+    AND (
+        (($process.name == logman.exe
+        OR $process.pe.original_file_name == Logman.exe))
```

Comment:
Could you use the hf `upcase` to validate the literal? what do you think?

Reply:
ditto